package.json

```diff
@@ -201,6 +222,16 @@
             "smartAutocomplete": false
           }
         },
+        "aurelia.relatedFiles": {

```

Comment:
Is it an idea to make these default settings arrays? 

And then:
- check the items of the array from the first to the last (if file exists)
- if so, switch to the file with that extension

Something like this:
```json
        "aurelia.relatedFiles": {
          "type": "object",
          "description": "Configure extensions for related files",
          "default": {
            "script": [".js", ".ts"],
            "style": [".less", ".sass", ".scss", ".styl", ".css"],
            "unit": [".spec.js", ".spec.ts"],
            "view": [".html"]
          }
        },
```
For example I don't use the same configuration as you do in any of my projects. That means for me out of the box it would not work. 

With the above, it will work for most people out of the box; and they still are able to adjust it.

It might only behave non-expected if you somehow only use SASS, but still have LESS files in your project (I assume that's during a move from LESS to SASS for example). Or if you let the SASS/LESS/Stylus processor generate CSS in the same directory and you removed the SASS/LESS/Stylus file, but did not cleanup the CSS file yet. But those are all Edge cases I think.

Reply:
You are right, the out of the box experience is important.
Will look into it!